Doc/library/curses.rst

```diff
@@ -1294,6 +1296,9 @@ Several constants are available to specify character cell attributes:
 | ``A_UNDERLINE``  | Underline mode.               |
 +------------------+-------------------------------+
 
+.. versionadded:: 3.7
+    ``A_ITALIC`` was added.
+
```

Comment:
I am not sure this doc change is suitable. This list lacks many options and seems only list some strict sysv options here.